python/docs/source/user_guide/pandas_on_spark/supported_pandas_api.rst

```diff
@@ -0,0 +1,803 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+=====================
+Supported pandas APIs
+=====================
+
+.. currentmodule:: pyspark.pandas
+
+The following table shows the pandas APIs that implemented or non-implemented from pandas API on Spark.
+
+Some pandas APIs do not implement full parameters, so the third column shows missing parameters for each API.
+
+If there is non-implemented pandas API or parameter you want, you can create an `Apache Spark JIRA <https://issues.apache.org/jira/projects/SPARK/summary>`_ to request or to contribute by your own.
+
+The API list is updated based on the `latest pandas official API reference <https://pandas.pydata.org/docs/reference/index.html#>`_.
+
+
+Supported I/O APIs
+------------------
+
+In general, the ``read_*`` APIs can be very expensive if the ``index_col`` parameter is not specified since it attaches the `default index <https://spark.apache.org/docs/latest/api/python/user_guide/pandas_on_spark/options.html#default-index-type>`_.
+
+In addition, the index will be lost if the ``index_col`` parameter is not specified for ``DataFrame.to_*`` APIs.
+
+====================== ======================= ====================================================================================================================================================================================================================================================================================================================================================================================
+API                    Implemented             Missing parameters
+====================== ======================= ====================================================================================================================================================================================================================================================================================================================================================================================
```

Comment:
@itholic can we add some newlines into the data cells, and make this bars less? Seems ugly in the test format

Reply:
Just reformed!